### PR TITLE
US11068 - Graph on Results Page

### PR DESCRIPTION
**Pull request by seancdavis**
Implements a graph on the results page using ngx-bootstrap. Graph has static values that Carabiners will later make dynamic.

The main discrepancy when compared to the designs is that there are more gridlines. From what I've been able to find, there is no way to manipulate these lines. If this becomes a blocker, I can look into some alternative solutions.

Corresponds to crdschurch/crds-styles#267.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +64,6 @@
     "ts-node": "~2.0.0",
     "tslint": "~5.7.0",
     "tslint-config-prettier": "^1.6.0",
-    "typescript": "2.5.3"
+    "typescript": "^2.4.2"
```
Comment (zarahzachz) [inline package.json]: Why did this need to be updated?

File: src/app/components/results-chart/results-chart.component.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  selector: "app-results-chart",
+  template: `
+    <div style="height: 250px;">
```
Comment (zarahzachz) [inline src/app/components/results-chart/results-chart.component.ts]: Are inline styles needed here?

Comment (seancdavis) [reply src/app/components/results-chart/results-chart.component.ts]: Good catch. I do need the explicit height on the container, but I meant to move this into a local style. I will do that.

File: package.json (hunk 1 above)
Comment (seancdavis) [reply package.json]: Found it. I updated this because of a command-line warning that typescript should be upgraded.

**Comment by seancdavis** [comment]
Ended up ditching ngx-charts because I had no control over the grid lines on smaller screens.

The current solution is a set of two SVG elements (one for desktop, one for mobile), where the paths within the bar shapes is drawn based on a value from 0-3, representing one of the four possible results.

**Pull request merged**